S21/ajayrazz99/wordChanger.py

```diff
@@ -0,0 +1,48 @@
+def LowerCaseConverter(word,lang):
```

Comment:
It would be better if a comment is added above the method to know what this functionality is for.

Reply:
yes, I agree. And the function's name was pretty much self explanatory but as you suggested adding a comment will surely help understand. Thanks!